data_model/src/lib.rs

```diff
@@ -281,97 +234,51 @@ pub mod parameter {
             PartialOrd,
             Ord,
             Hash,
-            Getters,
+            Constructor,
             FromStr,
+            Getters,
             Decode,
             Encode,
             Deserialize,
             Serialize,
             IntoSchema,
         )]
-        #[display(fmt = "{name}")]
         #[getset(get = "pub")]
         #[serde(transparent)]
         #[repr(transparent)]
         #[ffi_type(opaque)]
         pub struct ParameterId {
-            /// [`Name`] unique to a [`Parameter`].
+            /// Should be unique.
             pub name: Name,
         }
 
+        /// Executor configuration parameter.
+        ///
+        /// Must be defined in [`crate::executor::ExecutorDataModel`].
         #[derive(
-            Debug,
-            Display,
-            Clone,
-            Constructor,
-            IdEqOrdHash,
-            Decode,
-            Encode,
-            DeserializeFromStr,
-            SerializeDisplay,
-            IntoSchema,
+            Debug, Display, Clone, IdEqOrdHash, Decode, Encode, Deserialize, Serialize, IntoSchema,
         )]
-        #[display(fmt = "?{id}={val}")]
-        /// A chain-wide configuration parameter and its value.
+        #[display(fmt = "PARAMETER `{id}`")]
```

Comment:
Why did you remove the value from the display impl? I feel this might reduce usefulness of the log messages...

Reply:
I did it to unify `Parameter` and `Permission`: the latter one also displays only `id`, without `payload`.

Shall we then display payloads of both?

Reply:
I would prefer it this way, yeah. I don't think they will be _too large_, so it probably won't lead to any problems

Reply:
Done!